src/main/java/teammates/common/datatransfer/FeedbackContributionResponseDetails.java

```diff
@@ -45,23 +45,35 @@ public int getAnswer() {
     
     @Override
     public String getAnswerString() {
+        if (answer == -1) {
```

Comment:
Extract `-1` to a constant, something like `private static final int CONST_INVALID_RESPONSE = -1;`.